### ISSUE TITLE
Being taken to login screen when creating a loop template

### ISSUE THREAD

**Issue by 1118wp** [dudaster/ele-custom-skin#196]
I'm working on a staging site and want to create a few new archives.  The plugin seemed to install and activate just fine.  When I click on the "create/edit a loop template" button inside Elementor, I'm taken to a login screen for my main website.  The staging site is a subdomain of the main site.  Any idea what's happening?  I'm excited to use the plugin - it looks excellent!

**Comment by dudaster** [comment]
Every time?

**Comment by 1118wp** [comment]
Yes.  When I set up my staging site, I password protected it, so at first I thought that’s what was causing this.  I went in and removed the password protection and it’s still happening.

**Comment by dudaster** [comment]
And only with loop template does this?

**Comment by 1118wp** [comment]
Yes.  I was following the instructions given in this youtube video:

https://www.youtube.com/watch?v=ENZMamNQwNo <https://www.youtube.com/watch?v=ENZMamNQwNo>

**Comment by dudaster** [comment]
Try with another browser maybe it is a caching issue!
 Does this happens with every loop template?

**Comment by 1118wp** [comment]
I already tried clearing the cache both on the server and in my browser.  I’m doing my work in Chrome, version 81.0.4044.129.  Yes, it happens on every loop template.

**Comment by 1118wp** [comment]
I think I may have figured it out.  I went and checked my Elementor Pro licensing.  My staging site is indicating that it’s running Pro, but I went to my Elementor account on their site and it only shows my main site as being activated.  Elementor requires a separate license for staging sites and I do have the Pro Plus plan with 3 licenses available.  I then deactivated Pro on the staging site and activated it again.  I now get an error that won’t allow me to activate the license.  I then uninstalled and deleted both Elementor and Elementor Pro, cleared all caches, logged out, logged back in, and reinstalled from scratch.  Same result.  

Elementor no longer has controls in the licensing panel of “my account” to manually add the URL of my staging site to my license, so I’ve had to submit a support request. 

I also went to my live site and installed your plugin and it works fine there.  I think this is it.  I’ll let you know when I hear back from Elementor whether that cleared the issue up.  Seems to me I should be able to do some development with your plugin on the live site and then export the templates for use on the staging site when I have it working again.  Is there any reason that wouldn’t work?

**Comment by dudaster** [comment]
This is not normal. I think something is not working somewhere.

**Comment by 1118wp** [comment]
Okay.  I’ll get back to you when I hear from Elementor and am able to get Elementor Pro working properly on the staging site.  I’ll give it a test and keep you in the loop.

**Comment by dudaster** [comment]
If you can provide an url to take a look myself. You can also use the chat on dudaster.com

**Comment by 1118wp** [comment]
Elementor is fixed and that resolved the issue.  Thanks so much for your quick replies and concern.  I apologize for not at first realizing what was wrong.  Have a wonderful day!
